src/components/underkjenteAttesteringer/UnderkjenteAttesteringer.tsx

```diff
@@ -0,0 +1,83 @@
+import AlertStripe from 'nav-frontend-alertstriper';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import React from 'react';
+
+import { useI18n } from '~lib/hooks';
+import { Attestering, UnderkjennelseGrunn } from '~types/Behandling';
+import { formatDateTime } from '~utils/date/dateUtils';
+
+import messages from './underkjenteAttestering-nb';
+import styles from './underkjenteAttesteringer.module.less';
+
+const UnderkjenteAttesteringer = (props: { attesteringer: Attestering[] }) => {
+    const underkjenteAttesteringer = props.attesteringer.filter((att) => att.underkjennelse != null);
+
+    const { formatMessage } = useI18n({ messages });
+
+    const tidspunkter = underkjenteAttesteringer.map((a) => a.opprettet);
+    /* underkjente attesteringer har alltid grunn og kommentar */
+    /* eslint-disable @typescript-eslint/no-non-null-assertion */
+    const grunner = underkjenteAttesteringer.map((a) => a.underkjennelse!.grunn);
+    const kommentarer = underkjenteAttesteringer.map((a) => a.underkjennelse!.kommentar);
+    /*eslint-enable @typescript-eslint/no-non-null-assertion*/
```

Comment:
Heller map:a over underkjenteAttesteringer og vise informasjonen än å mappa over all informasjon i attesteringerna og sen mappa over de igen for å vise de.

Reply:
Det gjør bare at stylingen kommer stil å se wonky ut fordi hver rad har muligens ulik lengde. Noe som ikke kommer til å passe for alle de stedene der UnderkjenteAttesteringer brukes.

Reply:
Jeg skjønner ikke helt forklaringen din, men er enig med Aryan: virker ryddigere å ikke lage disse tre listene som tilfeldigvis (forhåpentligvis?) liner opp. Såvidt jeg klarer å se prøver vi her å få til en tre-kolonne-layout, noe som føles mer som en jobb for en tabell eller `display: grid`.

Reply:
Jepp, hvis du har en grid så kan du se till att alla kolonner får lika stor bredd. Så då gör det inget om 1a elementet er dobbelt så bred som 2a elementet f.eks.